views/addressbook/userguide.md

```diff
@@ -3,3 +3,153 @@ description: User Guide For Address Book Data Browser view
 ---
 
 # Address Book User Guide
+
+- [Introduction](#introduction)
+- [View](#view)
+- [Contacts ](#contacts-)
+  - [Search Contacts](#search-contacts)
+  - [Display a Contact](#display-a-contact)
+  - [Add a new Contact](#add-a-new-contact)
+  - [Delete an existing Contact](#delete-an-existing-contact)
+- [Groups](#groups)
+  - [New Group](#new-group)
+  - [Delete Group](#delete-group)
+  - [Group Sharing](#group-sharing)
+    - [Add](#add)
+    - [Adjust](#adjust)
+    - [Remove](#remove)
+- [Tools](#tools)
+
+
+## Introduction
+
+The Address Book view allows you to maintain your contacts, including grouping them together.
+
+## View
+To display an address book:
+1. In the [Data Browser](https://github.com/solid/userguide/README.md), navigate to the address book. 
```

Comment:
Hmm, I tried to find how to do this, but I couldn't find it. Perhaps add an instruction on how to navigate to the address book?

Reply:
I think perhaps it needs to be created first?

Reply:
You can have multiple address books that need to be created in advance. I'll include a note to that affect.